src/resolvers/mutations/accounts.ts

```diff
@@ -1,14 +1,22 @@
 import {UserInputError} from "apollo-server";
 
-const generateAccountTokens = async (root, {accountTokenInput} , {dataSources}) => {
+const generateAccountTokens = (root, {accountTokenInput} , {dataSources}) => {
     try {
         return dataSources.accountsAPI.generateAccountTokens( accountTokenInput )
     } catch (e) {
         throw new UserInputError("API returned with errors.", e)
     }
 }
+const accountAuthenticationSetting = (root, {accountTokenInput} , {dataSources}) => {
```

Comment:
this file is for mutation resolvers - maybe move this to a query resolver file.

Reply:
I missed these pieces, fix it.